examples/Channel.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace App;
+
+use Cmgmyr\Messenger\Traits\Messagable;
+use Illuminate\Database\Eloquent\Model;
+
+class Channel extends Model
```

Comment:
In which thing does this model differs from the threads model? If this should be the subject/object it would be better to use a name like `Car`, `House` or something like this.

Reply:
Ok sure the demo model could be named anything. The only point of it is that it's a firm thread based on an object, not a dynamic thread for any new user to user conversation

Reply:
Yes, that's why it would be great to choose a name that's not related to the repo topic.